lib/parameter_parser/search_parameter_parser.rb

```diff
@@ -299,6 +299,10 @@ def debug_options
         options[:new_weighting] = true
       when "explain"
         options[:explain] = true
+      when "include_withdrawn"
+        # Withdrawn content is excluded from regular searches but is useful for
+        # content audits
+        options[:withdrawn] = true
```

Comment:
Perhaps we could use `include_withdrawn` for the options hash too? That way it's consistent between the URL param and debug option.